README.md

```diff
@@ -79,13 +79,9 @@ The project is split in several Maven modules:
 
 * _engine_: The engine of the project. Most of the beef is here.
 
-* _infinispan_: Backend storing indexes in [Infinispan](http://infinispan.org/).
-
 * _integrationtest_: Integration tests with various technologies like WildFly, Spring or Karaf.
 Also includes performance tests.
 
-* _legacy_: Old Maven GAV kept for backward compatibility.
```

Comment:
To be sure: You removed it from the docs but still keep it around as an undocumented artefact until the next major?